.eslintrc

```diff
@@ -2,15 +2,14 @@
   "extends": [
     "plugin:@wordpress/eslint-plugin/recommended"
   ],
+  "globals": {
+    "describe": "readonly",
+    "expect": "readonly",
+    "it": "readonly"
+  },
```

Comment:
This is the main thing that fixed `npm run lint:js`.

The `overrides.files` weren't resolving right. I couldn't get it to point to `fse-studio`.

So I manually ignored these globals.

Reply:
Huh interesting - well nice approach! I suppose we can add any others if needed, when needed.

Reply:
Thanks! Yeah, we can add more as needed.